task_6_1.py

```diff
@@ -0,0 +1,12 @@
+TOTAL = 0
+with open('C:/Users/Andrii/Desktop/numbers.txt', 'r', encoding='utf-8') as inp,\
+     open('C:/Users/Andrii/Desktop/sum_numbers.txt', 'w', encoding='utf-8') as outp:
+    for line in inp:
+        try:
+            num = float(line)
+            TOTAL += num
+
+        except ValueError:
+            print(f'{line.format()} not a number!')
+    outp.write(format(TOTAL))
```

Comment:
why did you use format here?

Reply:
![image_2023-04-23_22-18-24](https://user-images.githubusercontent.com/113716040/233860387-f22ad645-df94-44f0-ab15-bcb3198d12a6.png)